src/eraftpb.rs

```diff
@@ -158,7 +158,7 @@ impl ::protobuf::Message for Entry {
         true
     }
 
-    fn merge_from(&mut self, is: &mut ::protobuf::CodedInputStream) -> ::protobuf::ProtobufResult<()> {
+    fn merge_from(&mut self, is: &mut ::protobuf::CodedInputStream<'_>) -> ::protobuf::ProtobufResult<()> {
```

Comment:
Will these still be reflected when we run `./generate-proto.sh`?

Reply:
They won't, but they are only idiomatic changes rather than breaking. To fix this we'd need to upgrade rust-protoc. Longer term, Prost will emit 2018 code when I eventually land https://github.com/danburkert/prost/pull/147

Reply:
@nrc Ok, is there a way to mechanize these changes when we do run `generate-proto.sh`?